source/core/ut_utils.pkb

```diff
@@ -384,5 +384,18 @@ create or replace package body ut_utils is
     commit;
   end;
 
+  function to_version(p_version_no varchar2) return t_version is
```

Comment:
IMO this function should raise an exception with proper message if an illegal version-no string is provided.